_data/accessibility-tests/file-input.yml

```diff
@@ -0,0 +1,103 @@
+    test_status: exception
+    status_details: Focus indicator is visible around the "Drag files" area as well as the "choose from folder" action link.
+    github_issue_number: 5616
+    github_issue_repo: uswds
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.7
+  - summary: File can be input with only a keyboard.
+    summary_additional: You can successfully upload a file using only a keyboard. No mouse or dragging action is required.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.5.7
+  - summary: Content doesn't change until the user takes an action to change it.
+    summary_additional: When you access the file input with a keyboard, content stays the same until you use keys to make changes.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.2
+# Screen reader tests
+  - summary: Screen reader announces file input.
+    summary_additional: When you navigate with arrow keys and the tab key while using a screen reader, you hear the prompt to "drag file here or choose from folder" and you hear any related hints or instructions.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.1
+  - summary: Fields and labels are read in the same order that they appear on the page.
+    summary_additional: When you use a screen reader and navigate through file input labels and fields, you hear each text label and field announced in the same order in which they appear on the page.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.2
+  - summary: File input field announces purpose.
+    summary_additional: When you tab into the file input using a screen reader, you hear messaging that helps you identify where you need to input the file for upload.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.5
+  - summary: Screen reader announces all labels.
+    summary_additional: When you use a screen reader and navigate to a file input using arrow and tab keys, you hear labels and status messaging for selection state.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.6
+  - summary: Screen readers identify file input boxes the same way when more than one is on a page.
+    summary_additional: When you tab to a file input on a page that has other file inputs, you will hear "file input" every time you tab into a new one.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.4
+  - summary: Screen reader announces incorrect file type.
+    summary_additional: When you upload a file in the input box, the screen reader announces if you have uploaded the wrong file type.
```

Comment:
@amycole501, @alex-hull 

**issue:** It feels like we should update this summary because the component doesn't currently provide a screen reader announcement when there is an incorrect file type. Feels we fail based on this summary. Wondering if it makes sense to move some of the info from 4.1.3 here, and then change 4.1.3 to include all status updates.

Reply:
@amyleadem Open to a conversation, but I think based on this criteria, it does fail. While it meets other error identification criteria, a screen reader user does not know that they have input an error. However! Our file input helps with some error prevention certain inputs. A user cannot upload a file that does not meet the specific format needs. But again, since there is no announcement to screen reader, I think this fails.

Reply:
@amyleadem @alex-hull Double checking that there isn't any edit to make here?

Reply:
@alex-hull @amyleadem Double checking that there is no edit that needs to be made for this?

Reply:
1. Move 3.3.1 from screen reader to General

2. **Error text indicates an incorrect file type**. When you drag an incorrect file type into the file upload box, text appears describing the error.

Reply:
I think this new summary works! 4.1.3 will be removed from this checklist from now.

Reply:
Resolved in [2a91b16](https://github.com/uswds/uswds-site/pull/2963/commits/2a91b1666fa6ea76c0d637a80fed290d33e9ed7b)